config/database.yml

```diff
@@ -20,8 +20,8 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: vladyslav
-  password: vlad1972
+  username: <%= ENV["USERNAME"] %>
+  password: <%= ENV["library_DATABASE_PASSWORD"] %>
```

Comment:
convention would be "DATABASE_USERNAME" and "DATABASE_PASSWORD"

also env vars should be ALL_CAPS, so `library_` part doesn't pass the vibe twice :D